Create ui-c8y-1023-30-2-disable-save-button-on-limits-tab-when-no-c-10851-graft.md (#4217)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-30-2-disable-save-button-on-limits-tab-when-no-c-10851-graft.md

---
date: ""

> title: disable save button on tenant Limits tab when no changes are made

Reviewer: ```suggestion
title: Disabled Save button on the Limits tab when nothing has been changed
```

> title: Disabled Save button on the Limits tab when nothing has been changed

Reviewer: ```suggestion
title: Save button on the Limits tab is only enabled when changes have been made
```

> title: Save button on the Limits tab is only enabled when changes have been made

Reviewer: ```suggestion
title: Save button on Limits tab only enabled when changes have been made
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62148
version: 1023.30.2
---

> The save button on the tenant Limits tab was enabled even when no changes were made. It is now disabled until the user modifies a value, preventing accidental saves of unchanged data.

Reviewer: ```suggestion
The **Save** button on the **Limits** tab in the tenant details was enabled even when no changes were made. It is now disabled until the user modifies a value, preventing accidental saving of unchanged data.
```
